project_task_category/models/project_task_category.py

```diff
@@ -11,4 +11,5 @@ class ProjectTask(models.Model):
     categ_id = fields.Many2one(
         'project.category',
         'Category',
+        domain=[('code', '=', 'categ_id')]
```

Comment:
@victormartinelicocorp Focusing on this particular change: I miss the point on the `domain`, and the `'categ_id'` hardcoded text. What is the point?